apps/web/components/AnchorNavigation/AnchorNavigation.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useMemo, useState } from 'react'
+import useScrollSpy from '../../hooks/useScrollSpy'
+import { Bullet } from '../Bullet/Bullet'
+import { Text, Box, FocusableBox, Stack } from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+import { useWindowSize, useIsomorphicLayoutEffect } from 'react-use'
+
+export interface AnchorNavigationProps {
+  title?: string
+  navigation: Array<{ id: string; text: string }>
+  position: 'left' | 'right'
+}
+
+export const AnchorNavigation = ({
+  title,
+  navigation,
+  position = 'left',
+}: AnchorNavigationProps) => {
+  const ids = useMemo(() => navigation.map((x) => x.id), [navigation])
+  const [activeId, navigate] = useScrollSpy(ids)
+  const [bulletRef, setBulletRef] = useState<HTMLElement>(null)
+  const { width } = useWindowSize()
+  const [isDesktop, setIsDesktop] = React.useState(false)
+
+  useIsomorphicLayoutEffect(() => {
+    if (width >= theme.breakpoints.lg) {
+      return setIsDesktop(true)
+    }
+    setIsDesktop(false)
+  }, [width])
+
+  return (
+    <Box
+      position="relative"
+      marginLeft={[0, 0, 0, 4]}
+      paddingBottom={[0, 0, 0, 6]}
+      paddingLeft={[3, 3, 4]}
+      borderLeftWidth={'standard'}
+      borderColor={'blue200'}
+      borderStyle="solid"
+    >
+      {bulletRef && isDesktop && (
+        <Bullet
+          top={bulletRef.offsetTop}
+          align={position === 'left' ? 'right' : 'left'}
+          color="blue"
+        />
+      )}
+
+      <Stack space={[1, 1, 2]}>
+        {Boolean(title) && (
+          <Box paddingBottom={1}>
+            <Text
+              variant="h4"
+              as="h2"
+              color={isDesktop ? 'blue600' : 'dark400'}
```

Comment:
I'm not sure if this is better than using duplicate text component for mobile and desktop, and maybe we want to change the color prop to a responsive one. what are your thoughts on this?

Reply:
this was actually an error in the design, it should be the same color in all screen sizes :D i'm going to revert this